City-of-Helsinki/drupal-asuntotuotanto
# SATT-52: Disable asu_admin theme and display archived button

### Description

Disable asu_admin theme and use hdbt_admin theme instead. Also go edit some project and check that archived button is showing perustiedot tab

### How to test it

- make fresh or make post-install
- make shell 
- drush uli --uid=85
- login as uid 85
- check that admin side looks ok
- open some project on edit form and check that you can see archived button next to locked and save
